Examples/WebViewExample/WebViewExample/ArticleView.swift

```diff
@@ -122,15 +140,32 @@ class ArticleView: UIView, WKNavigationDelegate {
             web.top == superview.top
             web.bottom == share.top
             share.bottom == superview.bottom
-            share.height == 88
+            if showShareButton {
+                share.height == 88
+                button.isHidden = false
+            } else {
+                share.height == 0
+                button.isHidden = true
+            }
         }
     }
 
     private func setupWebview() {
-        let request = URLRequest(url: planetData.url)
         webView.navigationDelegate = self
-        webView.load(request)
-
+        if planetData.url.scheme == "file" {
+            let baseURL = Bundle.main.bundleURL
+            let indexPath = baseURL.absoluteString + planetData.url.path
+            let indexURL = URL.init(string: indexPath)!
+            webView.loadFileURL(
+                indexURL,
+                allowingReadAccessTo: indexURL.deletingLastPathComponent()
+            )
+            showShareButton = false
+            setupConstraints()
```

Comment:
I think this will add new constraints each time. Usually I do something like:

```Swift
var constraintGroup = ConstraintGroup()

constraintGroup = constrain(view1, view2, replace: constraintGroup) {
    // etc.
}
```